tests/debugging_primitives_test.py

```diff
@@ -20,12 +20,12 @@
 import jax
 from jax import lax
 from jax import config
-from jax.experimental import maps
 from jax.experimental import pjit
 from jax.interpreters import pxla
 from jax._src import ad_checkpoint
 from jax._src import debugging
 from jax._src import dispatch
+from jax._src import maps
```

Comment:
Can this be `from jax._src.maps import xmap`?

Reply:
Done.